recipes/sphinx-autobuild/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2013, Jonathan Stoppani
```

Comment:
You don't need this file. It's included in the source from pypi so you should delete this file.